x11-terms/terminology/terminology-1.2.1-r1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson xdg-utils
+
+DESCRIPTION="Feature rich terminal emulator using the Enlightenment Foundation Libraries"
+HOMEPAGE="https://www.enlightenment.org/about-terminology"
+SRC_URI="https://download.enlightenment.org/rel/apps/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="nls +system-lz4"
+
+RDEPEND="
+	|| ( dev-libs/efl[egl] dev-libs/efl[opengl] )
+	|| ( dev-libs/efl[X] dev-libs/efl[wayland] )
+	>=dev-libs/efl-1.20.0[eet,fontconfig]
+	system-lz4? ( app-arch/lz4 )
+"
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+"
+
+src_prepare() {
+	default
+	xdg_environment_reset
+
+	if use system-lz4 ; then
```

Comment:
the patch is really tiny, apply it unconditionally

Reply:
and always use system-provided lz4 rather than bundled one? What if the build fails with system one? I wanted to apply it unconditionally but couldn't answer this question myself, so decided to make it a USE flag.

Reply:
but is there a reason to use bundled lz4? By standards and security reasons we try not to use bundled libs at all if possible.

Reply:
>By standards and security reasons we try not to use bundled libs at all if possible.

I'm aware :) 
My logic was, that it was probably bundled due to system lz4 causing breakages, and I didn't wish for that to happen. I'll gladly apply it unconditionally. 

And I talked about this with the author in IRC, and he's getting rid of it with terminology-1.3 (https://phab.enlightenment.org/T7328)

Anyway I'll remove the USE-case and rebase this PR in a minute.